source/fundamentals/builders/sort.txt

```diff
@@ -11,3 +11,326 @@ Sorts Builders
    :class: singlecol
 
 .. _sorts-builders:
+
+Overview
+--------
+
+In this guide, we show you how to specify **sort criteria** for your
+queries using **builders**.
+
+Sort criteria are the rules MongoDB uses to sort your data. Some
+examples of sort criteria are:
+
+* Smallest number to largest number
+* Earliest time of day to latest time of day
+* Alphabetical order by first name 
+
+The builder pattern allows you to create complex immutable objects
+separately from their representation. :doc:`Builders </fundamentals/builders/>` 
+are classes provided by the MongoDB Java driver that allow you to use
+the builder pattern to construct
+:java-docs:`Bson <apidocs/bson/org/bson/conversions/Bson.html>` objects.
+The builders provided by the MongoDB Java driver can improve the
+readability of your code.
+
+You should read this guide if you would like to:
+
+* Use the builder pattern to specify sort criteria for your queries.
+* Perform ascending sorts and descending sorts.
+* Combine sort criteria.
+* Sort on the text score of a
+  :manual:`text search </core/text-search-operators/>`.
+
+If you want to learn the fundamentals of sorting in the MongoDB Java
+driver, consider reading our
+:doc:`guide on sorting </fundamentals/crud/read-operations/sort/>`.
+
+The Sorts Class
+---------------
+
+The :java-docs:`Sorts </apidocs/mongodb-driver-core/com/mongodb/client/model/Sorts.html>`
+class is a builder that provides static factory methods for all sort criteria
+operators supported by MongoDB. All methods of ``Sorts`` return a
```

Comment:
Thanks for the note. I guess I have started defaulting to restating full referent rather than using pronouns, even in cases where it is less helpful. Going forward, I will try to be more critical in regards to using pronouns or restating referent and choosing the best option for the situation.